doc/source/cluster/ray-client.rst

```diff
@@ -17,7 +17,7 @@ By changing ``ray.init()`` to ``ray.client(...).connect()``, you can connect to
    import ray
 
    # Starting the Ray client. This connects to a remote Ray cluster.
-   ray.client("<head_node_host>:10001").connect()
+   ray.init("ray://<head_node_host>:10001").connect()
```

Comment:
do you still need `.connect()` here?

Reply:
Woops, no, not needed. Will fix in followup